tests/conftest.py

```diff
@@ -822,6 +823,15 @@ def cluster_teardown_finalizer():
     if deploy:
         # Deploy cluster
         deployer.deploy_cluster(log_cli_level)
+        # Workaround for #1777384 - enable container_use_cephfs on RHEL workers
```

Comment:
can you please raise a github ticket for workaround, we use that to revert when the workaround is not required. Thanks

Reply:
I've renamed the original ticket so it will be clear #1151